skins/cat17/src/app/lib/reducers/form_content.js

```diff
@@ -21,17 +21,22 @@ function trimValue( value ) {
 function forcePersonalDataForDirectDebit( state ) {
 	if ( state.paymentType === 'BEZ' && state.addressType === 'anonym' ) {
 		return objectAssign( {}, state, { addressType: 'person' } );
-	} else {
-		return state;
 	}
+	return state;
 }
 
 function forceAddressTypeForActiveMembership( state ) {
 	if ( state.membershipType === 'active' ) {
 		return objectAssign( {}, state, { addressType: 'person' } );
-	} else {
-		return state;
 	}
+	return state;
+}
+
+function forceOneTimePaymentForSofort( state ) {
```

Comment:
I know it's `Sofort Überweisung` but I think we agreed to refrain from German words in the code. 
I'd say `forceOneTimePaymentForSUB` goes more in that right direction.

Reply:
Yes, "Sofort" is the name of the Company/Payment type.